keyboards/mechwild/mokulua/mirrored/mirrored.c

```diff
@@ -28,7 +28,7 @@ bool encoder_update_kb(uint8_t index, bool clockwise) {
 #endif
 
 #ifdef OLED_ENABLE
-	oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
+	oled_rotation_t oled_init_user(oled_rotation_t rotation) {
```

Comment:
Please revert this change.  This should be the kb level function, NOT the user level. 
```suggestion
	oled_rotation_t oled_init_kb(oled_rotation_t rotation) {
```